Update subscribing-applications.md (#2974)
SoftwareAG/c8y-docs: content/enterprise-tenant/managing-tenants-bundle/subscribing-applications.md

---

### Overview

> The application concept of {{< product-c8y-iot >}} includes a basic application marketplace.

Reviewer: ```suggestion
{{< product-c8y-iot >}} features an application marketplace that allows tenants to access and manage various applications.
```

> Tenants can be subscribed to applications which have been deployed by their superior tenant ({{< management-tenant >}} or {{< enterprise-tenant >}}).
> 
> Via the Administration application, you can [grant application access to subtenants](/standard-tenant/managing-permissions/#application-access) and [subscribe to applications](#to-subscribe-an-application).
> 

Reviewer: ```suggestion
{{< management-tenant >}}s and {{< enterprise-tenant >}}s can control application access for their subtenants. Through the Administration application, you can:

- View and manage existing application subscriptions 
- [Subscribe subtenants to applications](#to-subscribe-an-application)
```

Author: @rahulc8y I don´t really get that. Subscribing/unsubscribing applications to subtenants is done in the tenant details as well as viewing and managing existing subscriptions. 
What can I do then on the application access tab under permissions?  /standard-tenant/managing-permissions/#application-access

Reviewer: In the proposed sentence i am referring to this section Cumulocity
Tenants>Subtenants > Applications, where the user can Subscribe to applications subtenants

![image](https://github.com/user-attachments/assets/80ebe511-bf2f-4db2-87e5-6835d2d6dc15)

Author: Yes, but the link you are using in "[View and manage existing application subscriptions] (/standard-tenant/managing-permissions/#application-access)" goes to 
![image](https://github.com/user-attachments/assets/f45d4cc1-9853-4f1f-b394-18f7bc4c53ca)
I wonder what is the use of this tab in the user details. We should link it here but I think the description of  the link is wrong as it is not about viewing and managing subscriptions.

Reviewer: Ok understood. in that case we can leave the existing content.

{{< c8y-admon-info >}}

> Tenants can also have their own applications which can also be subscribed to subtenants. See [Custom applications](/standard-tenant/ecosystem/#custom-applications) for details on how to add custom applications.

Reviewer: ```suggestion
Tenants can also create and deploy their own [Custom applications](/standard-tenant/ecosystem/#custom-applications), which can be made available to their subtenants. 
```

---

### Overview

{{< product-c8y-iot >}} features an application marketplace that allows tenants to access and manage various applications.

{{< management-tenant >}}s and {{< enterprise-tenant >}}s can control application access for their subtenants. In the **Applications** tab of a tenant, they can:

- [View and manage existing application subscriptions](to-view-subscribed-applications)

> - [Subscribe subtenants to applications](#to-subscribe-an-application)

Reviewer 2: I always thought its about subscribing an application to a subtenant and not about subscribing a subtenant to an application? Or can this be used as you want?

Author: This is used in a mixed way :-). But I also prefer to subscribe applications to tenants, will change it. What is your take, @rahulc8y ?

Reviewer: Agree, Subscribe applications to tenants seems to be correct.

> - [Subscribe subtenants to applications](#to-subscribe-an-application)

Author: ```suggestion
- [Subscribe applications to tenants](#to-subscribe-an-application)
```

- [Monitor the microservice status](#to-monitor-the-microservice-status)

For general information on applications as part of the {{< product-c8y-iot >}} ecosystem, refer to [Managing the ecosystem](standard-tenant/ecosystem/#managing-applications).

{{< c8y-admon-info >}}

> Tenants can also create and deploy their own [Custom applications](/standard-tenant/ecosystem/#custom-applications), which can be made available to their subtenants.

Author: ```suggestion
Tenants can also create and deploy their own [custom applications](/standard-tenant/ecosystem/#custom-applications), which can be made available to their subtenants.
```

{{< /c8y-admon-info >}}

### To view subscribed applications {#to-view-subscribed-applications}

> In the **Applications** tab of a tenant you can view all subscribed applications, subscribe tenants to applications or remove the applications from the tenant. By default, tenants will be subscribed to the standard {{< product-c8y-iot >}} applications.

Author: ```suggestion
In the **Applications** tab of a tenant you can view all subscribed applications, subscribe applications to tenants, or remove the applications from the tenant. By default, the [standard {{< product-c8y-iot >}} applications](/standard-tenant/ecosystem/#applications-subscribed-by-default) are subscribed to the tenant.
```

A tenant can have multiple available applications as displayed under **Available applications** but to use an application's functionality a subscription to the application must be established for the tenant. The list of subscribed applications is shown under **Subscribed applications**.

<img src="/images/users-guide/enterprise-tenant/et-subtenant-applications.png" alt="Subscribe tenant" style="max-width: 100%">

{{< c8y-admon-info >}}
Alternatively, you can retrieve the list of subscribed applications for a tenant by using the {{< product-c8y-iot >}} REST API to get [specific tenant information](https://{{< domain-c8y >}}/api/core/#tag/Tenant-API). The subscribed applications will be listed under the <code>applications</code> fragment.
{{< /c8y-admon-info >}}

#### To view the application details {#to-view-the-application-details}

Expanding an application entry in the **Available applications** list shows the following details:

> - Availability - One of MARKET, SHARED or PRIVATE

Author: Not sure if this list is complete. I could not find any explanations in the docs but perhaps I have overseen it? Same for the types below.

Reviewer 2: The list is complete

> - Type - One of HOSTED or MICROSERVIVE

Author: @reey How about this one?

Reviewer 2: There is in general also "EXTERNAL"

> - Type - One of HOSTED or MICROSERVIVE

Reviewer: ```suggestion
- Type - One of HOSTED, MICROSERVICE or EXTERNAL
```
